scripts/github_runner/setup_runner_host.sh

```diff
@@ -33,6 +36,13 @@ sudo apt-get install -y \
     jq \
     software-properties-common >> /dev/null
 
+if [ "${MULTIPASS}" == "multipass" ]; then
+	sudo apt-get -y install snapd >> /dev/null
+	sleep 10s
+	sudo snap install multipass >> /dev/null
+	exit 0
```

Comment:
why sleep and why exit 0? because simplicity in our bash scripts is preferable

Reply:
Exit 0 because nothing else has to be installed, so just to go out of the script (thought it would be more readable than a if-else). We can of course also go for a version where everything is installed without choice between multipass or docker.

sleep 10s because I had some issues with snap taking some time to start up and be ready for installation.

Reply:
Ok. Yes let's install everything without a choice

Reply:
Since it just happens sporadically, I also removed the sleep, but one can see if it fails, and just restart the script.

Reply:
I think it's better to leave the sleep if it's more stable this way

Reply:
so what's about the sleep? restarting is not a good idea because self-hosted runners are already not quite stable

Reply:
Yes, it is now back in.